app/furniture/marketplace/delivery_areas/index.html.erb

```diff
@@ -1,18 +1,20 @@
 <%- breadcrumb(:marketplace_delivery_areas, marketplace) %>
 
-<section class="max-w-2xl self-stretch mx-auto">
-  <main>
-    <div class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
-    <% delivery_areas.each do |area| %>
-      <%= render CardComponent.new(classes: "py-2") do %>
-        <%= link_to area.location(:edit) do %>
-          <%= area.label %>: <%= number_to_currency(area.price) %>
-        <%- end %>
+<%= render Marketplace::ManagementComponent.new(marketplace: marketplace) do %>
+  <section class="max-w-2xl self-stretch mx-auto">
+    <main>
+      <div class="mt-3 grid grid-cols-1 gap-5 sm:gap-6 sm:grid-cols-2 lg:grid-cols-4">
+      <% delivery_areas.each do |area| %>
+        <%= render CardComponent.new(classes: "py-2") do %>
+          <%= link_to area.location(:edit) do %>
+            <%= area.label %>: <%= number_to_currency(area.price) %>
+          <%- end %>
+        <% end %>
       <% end %>
-    <% end %>
-  </main>
+    </main>
 
-  <footer class="text-center">
-    <%= link_to(t("marketplace.delivery_areas.new"), marketplace.location(:new, child: :delivery_area)) %>
-  </footer>
-</section>
\ No newline at end of file
+    <footer class="text-center">
+      <%= link_to(t("marketplace.delivery_areas.new"), marketplace.location(:new, child: :delivery_area)) %>
```

Comment:
We've been using the convention `controller.action.link_to` for our lookups, i.e. "marketplace.delivery_areas.new.link_to`

This way they live next to the `marketplace.delivery_areas.new.success` and `marketplace.delivery_areas.new.failure` items

Reply:
I'll address in my follow-up.

Reply:
FWIW, this was pre-existing, I just wrapped this whole thing in the new component.